mobile/src/reducers/organization/organization.epics.ts

```diff
@@ -50,3 +51,24 @@ export const loadEmployeesForDepartmentEpic$ = (action$: ActionsObservable<LoadE
         .mergeAll()
         .flatMap(x => x.map(loadEmployeeFinished))
         .catch((e: Error) => Observable.of(loadFailedError(e.message)));
+
+export const loadFeedsEpic$ = (action$: ActionsObservable<LoadFeeds>) =>
+    action$.ofType('LOAD_FEEDS')
+        .switchMap(x => ajaxGetJSON(`${url}/feeds/messages`))
+        .map(x => deserializeArray(x as any, Feed))
+        .map(x => loadFeedsFinished(x))
+        .catch(e => Observable.of(loadFailedError(e.message)));
+
+export const loadFeedsFinishedEpic$ = (action$: ActionsObservable<LoadFeedsFinished>) =>
+    action$.ofType('LOAD_FEEDS_FINISHED')
+        .flatMap(x => x.feeds.map(feed => loadEmployeeForFeed(feed.employeeId)));
```

Comment:
why not use `loadEmployee` instead?

Reply:
Because I couldn't make it work

Reply:
what exactly didn't work? It's supposed to work.